deps/node/deps/uv/src/threadpool.c

```diff
@@ -55,6 +55,7 @@ static void uv__cancelled(struct uv__work* w) {
  * never holds the global mutex and the loop-local mutex at the same time.
  */
 static void worker(void* arg) {
+  uv_thread_setname("lwnode::uv");
```

Comment:
```suggestion
static void worker(void* arg) {
```

Reply:
IMO, no need to name at this level. Naming for debugging should be set at the higher level that uses uv.